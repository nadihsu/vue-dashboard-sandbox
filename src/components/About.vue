<template>
  <div>
    <p>This is about page</p>
  </div>
</template>
