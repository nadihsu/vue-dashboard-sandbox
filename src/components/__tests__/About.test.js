import { describe, it, expect } from 'vitest';
import { mount } from '@vue/test-utils';
import About from '../About.vue';

describe('About Page', () => {
  it('基本', () => {
    const render = mount(About);

    expect(render.text()).toContain('This is about page');
    expect(render.html()).toMatchSnapshot();
  });
});
