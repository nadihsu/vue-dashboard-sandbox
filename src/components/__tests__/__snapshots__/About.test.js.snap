// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`About Page > 基本 1`] = `
"<div>
  <p>This is about page</p>
</div>"
`;
